site/en/docs/search_reranking_using_embeddings.ipynb

```diff
@@ -31,19 +31,6 @@
         "# limitations under the License."
```

Comment:
<p>&gt;Here you don't want the model changing the n_topics, so a simple wrapper hides it from the model.</p><p> </p><p>Just an idea, but it looks like we only ever use the default value in this guide - you could extract to a fn-local var so you don't have to wrap it, but it's still clear how a user could change it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>+1</p>